ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<blockquote>model_name = "[your-model-id]"</blockquote><p>Can we rewrite this to model_resource_name = "[your-model-resource-name]"? It's a bit confusing because "name" and "id" are both used. I initially used the wrong value (hit an error later on), and it took a little bit of troubleshooting to understand this was the issue. Also, the model.list from above labels this as the "resource name"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Done.</p>